packages/faustwp-core/src/index.ts

```diff
@@ -41,8 +41,14 @@ import {
   ToolbarSubmenu,
   ToolbarSubmenuWrapper,
 } from './components/Toolbar/index.js';
+import {
+  flatListToHierarchical,
+  ListNode,
+} from './utils/flatListToHierarchical.js';
 
 export {
+  flatListToHierarchical,
+  ListNode,
```

Comment:
Do we need to expose `ListNode` here?

Reply:
Yes

Reply:
Yes. We had to make it publicly exposed through /core. Previously it had Node and that was clashing with some other namespace so we created and exposed this.

Reply:
@matthewguywright What is it used for? Isn't it just a type for an internal param?

If we expose it in the main export we'll need to support it indefinitely since it'll become apart of our public API.